### ISSUE TITLE
Double entries

### ISSUE THREAD

**Issue by kelylah** [krampstudio/grunt-jsdoc#138]
When I'm generating documentation using grunt plugin I get double entries like this:
![image](https://cloud.githubusercontent.com/assets/3965339/8078043/fde1c91a-0f5a-11e5-88f1-07472473729d.png)

When I'm using clearly jsdoc than every thing is ok.

My configuration
```js
jsdoc: {
			dist: {
				src: ['./**/*.js', '!./**/strings.js', '!./**/setting/*', '!./general-doc/site/**/*', '!./node_modules/**/*', '!./api-doc/**/*', '!./dev-doc/**/*'],
				dest: './api-doc/',
				options: {
					configure: './jsdoc_config.json'
				}
			}
		}
```

and *jsdoc_config.json*
```js
{
    "tags": {
        "allowUnknownTags": true
    },
    "source": {
				"include": ["./", "./README.md"],
				"exclude": ["api-doc", "general_doc", "node_modules"],
        "includePattern": ".+\\.js(doc)?$",
        "excludePattern": "(^|\\/|\\\\)\\.|strings.js"
    },
    "plugins": [],
    "templates": {
        "cleverLinks": false,
        "monospaceLinks": false,
        "default": {
            "outputSourceFiles": false
        },
				"navType": "vertical",
				"linenums": true,
				"collapseSymbols": true,
				"copyright": ".:EPL DEV TEAM:. (C)Esri Polska 2015",
				"systemName": "WAB Widgets",
				"theme": "spacelab"
    },
		"opts": {
			"template": "./node_modules/ink-docstrap/template",
			"encoding": "utf8",             
			"destination": "./api-doc/",    
			"recurse": true,                
			"private": true
		}
}
```

**Comment by krampstudio** [comment]
Isn't it because the `source` of the config file and the `src` propery of the grunt plugin add the same file 2 times ?

**Comment by kelylah** [comment]
Hi thanks for answer.

When I removed *include* from *jsdoc_config.json* than it helps.Unfortunately I can't remove it from grunt task because I get warning:
```
Running "jsdoc:dist" (jsdoc) task
Warning: Cannot call method 'indexOf' of undefined Use --force to continue.

Aborted due to warnings.
```

It isn't comfortable to make changes in config depending you want grunt task or normal jsdoc generation.
I've prepared two files with jsdoc configuration and that is good enought walkaround.
